google/resource_compute_disk.go

```diff
@@ -279,12 +287,14 @@ func resourceComputeDiskRead(d *schema.ResourceData, meta interface{}) error {
 	}
 
 	var disk *compute.Disk
-	if zone, ok := d.GetOk("zone"); ok {
+	if zone, err := getZone(d, config); zone != "" && err == nil {
```

Comment:
If the zone is empty, the `err` returned by  `getZone` should not be nil and therefor, the `zone != nil` is redundant

Reply:
Ah, shoot, I thought that was idiomatic because of the way that .Get() works in Schema.  :)  I'll fix it.